data/json/bionics.json

```diff
@@ -1129,6 +1129,7 @@
     "enchantments": [
       {
         "condition": "ACTIVE",
+        "//": "TODO: Remove MAP_MEMORY after 0.G Stable, as map memory is now permanent",
```

Comment:
Why not removing it now? It already doesn't do anything since map memory is already permanent